AddSevenPS.sln.DotSettings.user

```diff
@@ -0,0 +1,12 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+<<<<<<< HEAD
+	<s:String x:Key="/Default/Environment/AssemblyExplorer/XmlDocument/@EntryValue">&lt;AssemblyExplorer&gt;&#xD;
+  &lt;Assembly Path="C:\Users\Aryan Sharma\RiderProjects\AddSevenPS\AddSevenPS\bin\Debug\AddSevenPS.dll" /&gt;&#xD;
+&lt;/AssemblyExplorer&gt;</s:String></wpf:ResourceDictionary>
+=======
+	<s:String x:Key="/Default/Environment/UnitTesting/UnitTestSessionStore/Sessions/=fcd77302_002D8301_002D4c7b_002D82e7_002Dfc99bf0260c9/@EntryIndexedValue">&lt;SessionState ContinuousTestingMode="0" IsActive="True" Name=" AddSeven.addSeven -&amp;gt; &amp;quot; 1 =&amp;gt; 8&amp;quot; " xmlns="urn:schemas-jetbrains-com:jetbrains-ut-session"&gt;&#xD;
+  &lt;TestAncestor&gt;&#xD;
+    &lt;TestId&gt;xUnit::49984BB5-D987-4043-A6B6-4C09CCD60730::net6.0::AddSevenPS.LibraryTest&lt;/TestId&gt;&#xD;
+  &lt;/TestAncestor&gt;&#xD;
+&lt;/SessionState&gt;</s:String></wpf:ResourceDictionary>
+>>>>>>> origin/develop
```

Comment:
@Aryan-sharma11 , this indicates an unclean merge process. 🔴

Reply:
> @Aryan-sharma11 , this indicates an unclean merge process. 🔴

Is this file necessary or should I add this too in .gitignore ??

Reply:
Yes this also resolved in commit 676f1bcfb910d79c27c7ea4ef9a745a8c59e6327